geodata/search.py

```diff
@@ -90,7 +90,8 @@ def search_near_point_filter(params):
 def search_near_point(request, *args, **kwargs):
     params = request.query_params
     results = SpatialEntry.objects.filter(search_near_point_filter(params))
-    return JsonResponse(serializers.SpatialEntrySerializer(results, many=True).data, safe=False)
+    serializer = utility.create_serializer(SpatialEntry)
```

Comment:
We probably don't want to create a class each time this function is called.  If the serializers are generated once (on import, for instance), then they could still be referenced by name as before.